run_dir/static/css/status_b5.css

```diff
@@ -933,3 +933,96 @@ th.sort{
   cursor: pointer;
   margin-right: 4px;
 }
+
+
+/*
+=== User Agreement ===
+*/
+
+@page {
+  size: A4;
+  margin-top: 165px;
+  text-align: justify;
+  text-justify: justify;
+  text-align-last: left;
+  orphans: 4;
+  widows: 4;
+
+  @top-left: {
+    content: element(titleImgRunning);
+  }
+  @bottom-right {
+    content: "Page " counter(page) " of " counter(pages);
+    border-top:1px solid;
+    text-align-last: right;
+ }
+}
+
+.dl-horizontal-quote-details{
+  background-color: rgba(0, 0, 0, 0.15);
+}
+
+.dl-horizontal-quote-details > dt{
+  float: left;
+  text-align: left;
+  width: 160px;
+
+}
+
+.dl-horizontal-quote-details> dd{
+  margin-left: 180px;
+  margin-bottom: 0px;
+  padding: .375rem .375rem;
+}
+
+.dl-horizontal-quote-price > dt{
+  float: left;
+  text-align: left;
+}
+
+.dl-horizontal-quote-price> dd{
+  margin-left: 180px;
+  margin-bottom: 0px;
+  text-align-last: right;
+}
+
+.dl-horizontal-quote-details> dd > p{
+  margin-bottom: 0.3rem !important;
+}
+
+.title-img {
```

Comment:
Maybe this could be slightly less generic? So that it doesn't get changed by accident later on

Reply:
Done!